commcare/src/main/resources/webapp/partials/settings.html

```diff
@@ -11,7 +11,7 @@
              <a ng-click="addConfig()" type="button" class="form-group btn btn-success">
                  <i class="fa fa-plus"></i> {{msg('commcare.addConfig')}}
              </a>
-             <a ng-if="selectedConfig" ng-disabled="!saveAllowed()" ng-click="saveConfig()" type="button" class="btn btn-primary">
+             <a ng-if="selectedConfig" ng-disabled="!saveAllowed()" ng-click="saveAllowed() && saveConfig()" type="button" class="btn btn-primary">
```

Comment:
If ng-disabled does not work on anchor links, we should either change them to buttons, or adjust the ng-click logic in the remaining buttons as well.